cli/src/pcluster/resources/compute_node/user_data.sh

```diff
@@ -149,10 +149,23 @@ write_files:
       [ -f /etc/profile.d/proxy.sh ] && . /etc/profile.d/proxy.sh
       custom_cookbook=${CustomChefCookbook}
       export _region=${AWS::Region}
+
+      # The default curl setting will timeout the connect attempt at 5 minutes, so
+      # having the retry count set to 3 will result in 15m to failure if there is
+      # no internet connectivity from the compute node.
+      cookbook_url_retry_strategy=--retry 3
```

Comment:
It looks like we are not changing the numbers of retry in this PR?  The original retry is also 3 times